foundation_cms/templates/patterns/components/footer.html

```diff
@@ -34,31 +34,38 @@
         </ul>
       </div>
       <div class="site-footer-ns__grid-cell site-footer-ns__cta">
-        <button type="button">Button text</button>
+        <button type="button" class="site-footer-ns__button site-footer-ns__button--cta">
+          <span>
+            Donate now
+          </span>
+          <svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M7.74 1.36019L13.63 7.25019H0.5V8.75019H13.63L7.74 14.6402L8.8 15.7002L16.5 8.00019L8.8 0.300186L7.74 1.36019Z" fill="#161616"/>
+          </svg>
+        </button>
       </div>
     </div>
-    <div class="site-footer-ns__legals">
+    <div class="site-footer-ns__legals body-text-xsmall">
       <div class="site-footer-ns__grid-cell">
-        <p>Mozilla Foundation is a global non-profit and parent of the <a href="#">Mozilla Corporation</a>. Most content available under a <a href="#">Creative Commons license</a>.</p>
+        <p>Mozilla Foundation is a global non-profit and parent of the <a href="#" class="site-footer-ns__link site-footer-ns__link--legal">Mozilla Corporation</a>. Most content available under a <a href="#" class="site-footer-ns__link site-footer-ns__link--legal">Creative Commons license</a>.</p>
       </div>
       <div class="site-footer-ns__grid-cell">
         <ul class="social-links">
           <li>
-            <a href="https://bsky.app/profile/mozilla.org" aria-label="Bluesky">
+            <a href="https://bsky.app/profile/mozilla.org" class="site-footer-ns__social-link" aria-label="Bluesky">
               <i class="icon-bluesky">
                 <svg width="24" height="23" viewBox="0 0 24 23" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M12 10.2734C10.913 8.15945 7.954 4.22045 5.202 2.27845C2.566 0.417447 1.561 0.739447 0.902 1.03845C0.139 1.38145 0 2.55345 0 3.24145C0 3.93145 0.378 8.89145 0.624 9.72045C1.439 12.4564 4.337 13.3804 7.007 13.0844C7.143 13.0644 7.282 13.0454 7.422 13.0284C7.284 13.0504 7.146 13.0684 7.007 13.0844C3.095 13.6644 -0.38 15.0894 4.177 20.1624C9.19 25.3524 11.047 19.0494 12 15.8544C12.953 19.0494 14.05 25.1254 19.733 20.1624C24 15.8544 20.905 13.6644 16.993 13.0844C16.8542 13.0691 16.7159 13.0504 16.578 13.0284C16.718 13.0454 16.857 13.0644 16.993 13.0844C19.663 13.3814 22.561 12.4564 23.376 9.72045C23.622 8.89245 24 3.93045 24 3.24245C24 2.55245 23.861 1.38145 23.098 1.03645C22.439 0.738447 21.434 0.416447 18.798 2.27645C16.046 4.22145 13.087 8.16045 12 10.2734Z" fill="white"/></svg>
```

Comment:
Are we animating the SVG itself here? If not, could we link to the file instead of inlining it?

Reply:
Same with other SVG files used in this file. Let's link to the SVG files directly.